### DISCUSSION TITLE
Salvar como .xlsx

### DISCUSSION THREAD

**Discussion by oilujrasec** [Daniel09Fernandes/Component_OpenOffice#76] [General]
Uso o office 2021.

Coloquei formula que totaliza os valores de uma coluna em outra ABA.

No Libreoffice coloquei assim  **=$Detalhamento.C2:C21**, perfeito!

Uso o comando SaveFile('c:\planilja.xlsx') salvando para excel, e fica assim:

**=@Detalhamento!C2:C21** que não funciona, sendo creio eu que deveria ficar assim: **=SOMA(Detalhamento!C2:C21)**

Necessita configurar mais alguma coisa para funcionar?

**Comment by Daniel09Fernandes** [comment]
Bom dia Amigo, 

Realizei um teste aqui, da seguinte forma 
Na célula que ira totalizar, informa assim '=SOMA(Detalhamento.C2:C22)' 
Ao salvar como .xlsx estará compatível com o Excel. 

![image](https://github.com/user-attachments/assets/f2b6b9e6-05a9-4df8-ada3-0d0f4b110e43)

**Reply by oilujrasec** [reply]
Eu faço o preenchimento no código, e tentei de outras formas. nesse teu exemplo:

OpenofficeCalc.setFormula(18,'F',  '=SOMA(Detalhamento.C2:C' + _linha.ToString + ')');

Realmente não dá certo.

No resultado final.... Ele coloca esse @
![image](https://github.com/user-attachments/assets/f8d546c0-a0e6-45a2-8e01-d795f976db4c)

**Comment by Daniel09Fernandes** [comment]
Realizei uma correção no código, pode atualizar o componente e testar novamente? 

Salvando a mesma formula em xlsx, abrindo com o Libreoffice
![image](https://github.com/user-attachments/assets/6a38a7c8-3988-48d4-8550-a4f5d561ee26)

Aqui pelo Excel
![image](https://github.com/user-attachments/assets/4f642c21-34fc-4ccd-aa43-ce3735abd5c5)

**Reply by oilujrasec** [reply]
Show de bola amigo, funfo lindo, vlw
